src/main/java/org/fundacionjala/fc/client/Authentication.java

```diff
@@ -0,0 +1,56 @@
+package org.fundacionjala.fc.client;
+
+import io.restassured.RestAssured;
+import io.restassured.response.Response;
+import io.restassured.specification.RequestSpecification;
+import org.apache.http.entity.ContentType;
+
+public class Authentication {
```

Comment:
Please consider converting this class to a utility class, so, please add the `final` non-access modifier. and change the name to `ReqSpecUtils`.
```java
public final class ReqSpecUtils {
```